examples/with-sentry/README.md

```diff
@@ -1,55 +1,72 @@
 
-An example showing use of [Sentry](https://sentry.io) to catch & report errors on both client + server side.
+# Sentry
 
-## How to use
+This is a example showing how to use [Sentry](https://sentry.io) to catch & report errors on both client + server side.
+
+- `_app.js` renders on both the server and client. It initializes Sentry to catch any unhandled exceptions
+- `_error.js` is rendered by Next.js while handling certain types of exceptions for you. It is overridden so those exceptions can be passed along to Sentry
+- `next.config.js` enables source maps in production for Sentry and swaps out `@sentry/node` for `@sentry/browser` when building the client bundle
+
+## How To Use
 
 ### Using `create-next-app`
 
 Execute [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app) with [npm](https://docs.npmjs.com/cli/init) or [Yarn](https://yarnpkg.com/lang/en/docs/cli/create/) to bootstrap the example:
 
 ```bash
-npx create-next-app --example with-sentry with-sentry-app
+npx create-next-app --example with-sentry with-sentry
 # or
-yarn create next-app --example with-sentry with-sentry-app
+yarn create next-app --example with-sentry with-sentry
 ```
 
-### Download manually
+### Download Manually
 
 Download the example:
 
-Install it and run:
-
-**npm**
-
 ```bash
-npm install
-npm run dev
+curl https://codeload.github.com/vercel/next.js/tar.gz/canary | tar -xz --strip=2 next.js-canary/examples/with-sentry
+cd with-sentry
 ```
 
-**yarn**
+Install it and run:
 
 ```bash
+npm install
+npm run dev
+# or
 yarn
 yarn dev
 ```
 
 Deploy it to the cloud with [Vercel](https://vercel.com/import?filter=next.js&utm_source=github&utm_medium=readme&utm_campaign=next-example) ([Documentation](https://nextjs.org/docs/deployment)).
 
-### Configuration
+## Notes
 
-You will need a _Sentry DSN_ for your project. You can get it from the Settings of your Project, in **Client Keys (DSN)**, and copy the string labeled **DSN (Public)**.
+- By default, neither sourcemaps nor error tracking is enabled in development mode (see Configuration).
+- When enabled in development mode, error handling [works differently than in production](https://nextjs.org/docs#custom-error-handling) as `_error.js` is never actually called.
+- The build output will contain warning about unhandled Promise rejections. This caused by the test pages, and is expected.
+- The version of `@zeit/next-source-maps` (`0.0.4-canary.1`) is important and must be specified since it is not yet the default. Otherwise [source maps will not be generated for the server](https://github.com/zeit/next-plugins/issues/377).
+- Both `@zeit/next-source-maps` and `@sentry/webpack-plugin` are added to dependencies (rather than `devDependencies`) is because if used with SSR, these plugins are used during production for generating the source-maps and sending them to sentry.
```

Comment:
```suggestion
- Both `@zeit/next-source-maps` and `@sentry/webpack-plugin` are added to dependencies (rather than `devDependencies`) because if used with SSR, these plugins are used during production for generating the source-maps and sending them to sentry.
```